Initium-Odp/src/com/universeprojects/miniup/server/controllers/ViewItemController.java

```diff
@@ -683,6 +683,16 @@ else if(ownerHtmlSpecified == false && GameUtils.enumEquals(scriptType, ODPDBAcc
 		if (field!=null && field.toString().trim().equals("")==false)
 			itemMap.put("description", field.toString());
 		
+		//If we have explicitely allowed charges to be visible on this item, display them.
+		Boolean chargesVisible = (boolean) item.getProperty("chargesVisible");

```

Comment:
this will cause  an error if charchesVisible is null. You have to cast it as a "Boolean", not "boolean". Boolean can be null, but boolean is a primitive and cannot be null.